src/foam/nanos/hello/tests.jrl

```diff
@@ -0,0 +1,2 @@
+p({"class":"foam.nanos.hello.HelloTest","id":"HelloTest"})
+p({"class":"foam.nanos.test.Test","id":"HelloTestJS","server":false,"status":2,"code":"var hello = foam.nanos.hello.ClientHelloService.create({\ndelegate: foam.box.HTTPBox.create({\nurl: 'service/hello'\n}, x)\n}, x);\nhello.hello(\"JS\").then(function(msg) { foam.assert(msg === (\"helloJS\"), \"ClientHelloService succeed\") });"})
```

Comment:
Minor detail but it's also good to end files with a newline, especially line delimited files like journals.

 You can see that github is warning about that with the little symbols at the end of the diff here and in HelloService.js

Ideally you should configure your editor to do this automatically for you.